src/test/java/org/fcrepo/importexport/integration/ExporterIT.java

```diff
@@ -317,6 +321,44 @@ public void testExportCustomPredicatesInbound() throws Exception {
                 createResource(obj1.toString()),
                 createProperty("http://example.org/custom"),
                 createResource(col1.toString())));
+        assertFalse(collectionModel.contains(
+                createResource(other.toString()),
+                createProperty("http://example.org/other"),
+                createResource(col1.toString())));
+    }
+
+    @Test
+    public void testExportInboundFiles() throws Exception {
```

Comment:
@awoods This test demonstrates a fix to the bug in d34b5771ff4abeb6ecc52fa1fa528b0117ab1b9f — that I wrote up in [fcrepo-2500](https://jira.duraspace.org/browse/FCREPO-2500) — namely that the first pass at fixing inbound references was stripping all binary description properties (I was seeing a turtle file with only namespace declarations in it).

Would it be helpful to move this commit to its own commit?